src/software/thunderscope/thunderscope.py

```diff
@@ -148,6 +148,21 @@ def __init__(
         self.window.setCentralWidget(self.tabs)
         self.window.setWindowTitle("Thunderscope")
 
+        self.is_playing = True
+
+        simulationToolBar = self.window.addToolBar("simulation")
+
+        pixmapi_pause = QStyle.StandardPixmap.SP_MediaStop
```

Comment:
Would a pause icon be better suited here? `SP_MediaPause`
A stop icon might be confused for halting/exiting the simulation